mitmproxy/optmanager.py

```diff
@@ -416,6 +416,35 @@ def dump_defaults(opts):
     return ruamel.yaml.round_trip_dump(s)
 
 
+def dump_dicts(opts):
+    """
+        Dumps the options into a list of dict object.
+
+        Return: A list like: [ { name: "anticache", type: "bool", default: false, value: true, help: "help text"} ]
+    """
+    options_list = []
+    for k in sorted(opts.keys()):
+        o = opts._options[k]
+        if o.typespec in (str, int, bool):
```

Comment:
Move this typespec to string conversion into its own function in mitmproxy/utils/typecheck? :)